### PR TITLE
Update some things: CONST, Logging, robustness, SSL support

### PR DESCRIPTION
**Pull request by ue71603**
done first round

### CODE REVIEW

**Comment by ue71603** [comment]
@skinkie  If you can have a look. It is now an example with train spliting.

The total cost  should (with half-price subscription) be:
image

Otherwise
second 25
first = 25+19
discount ticekts: 22.40

**Comment by skinkie** [comment]
Should the .idea folder be included?

**Comment by ue71603** [comment]
will be excluded.

**Comment by skinkie** [comment]
git rm .idea/* ?

**Comment by ue71603** [comment]
@skinkie  does this change work for you? Then we could squashmerge it into master and remove this branch.

**Comment by skinkie** [comment]
I want to see if we can update the extension code.

**Comment by ue71603** [comment]
good idea. It looked so C... (and I should probably used break's to speed it up by 50%

File: logger.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import datetime
+
+# writing debugging files and specific points
+def log(filename,xml):
+    if DEBUGGING:
+        open(filename, 'w').write(xml)
+
+# writing which origin/destination was asked from NOVA
+def log_entry(origin, destination, starttime,stoptime):
+    logger = logging.getLogger('my_logger')
```
[the remark this answers is not in the document]
Comment (ue71603) [reply logger.py:15]: @skinkie  you did that , but it seems you don't log anything anymore. You removed that?  Where could we have for each request to NOVA have Origin/Destinat/Start time/Arrival time? I find it hard to add it currently.

[the remark this answers is not in the document]
Comment (ue71603) [reply logger.py:15]: log_entry seesm to be no longer used. But the logger you defined should be.

File: map_ojp_to_nova.py [hunk 2]
```diff
@@ -75,24 +80,26 @@ def map_fare_request_to_nova_request(ojp: Ojp, age: int=30) -> PreisAuskunftServ
...
+                                                                          geschaefts_prozess_id="1781786f-57ba-4e9a-bc29-287e2aa97f9a"),
+                                                                      angebots_filter=[TaxonomieFilter(
+                                                                          produkt_taxonomie="Basistaxonomie",
+                                                                          taxonomie_klasse_pfad=[TaxonomieKlassePfad(
+                                                                              klassen_name="Einzelbillette")])],
+                                                                      reisender=[ReisendenInfoPreisAuskunft(alter=age,
+                                                                                                            externe_reisenden_referenz_id="1234",
+                                                                                                            reisenden_typ=ReisendenTypCode.PERSON)],
+                                                                      verbindung=verbindungen
```
Comment (ue71603) [inline map_ojp_to_nova.py:101]: These are all defined parameters. Should we move them to configuration?

Comment (skinkie) [reply map_ojp_to_nova.py:101]: If you think it is useful.

Comment (ue71603) [reply map_ojp_to_nova.py:101]: I will think about it

**Pull request merged**